contracts/Marionette.sol

```diff
@@ -137,7 +150,15 @@ contract Marionette is IMarionette, AccessControlEnumerableUpgradeable {
 
         if (target.isContract()) {
             if (data.length >= 4) {
-                return target.functionCallWithValue(data, value);
+                if (_getHashedSelector(data) == _multiSendHashed()) {
```

Comment:
Is it possible to compare func signature (bytes4) instead of hash of func signature (bytes32)